src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseActionsResource.java

```diff
@@ -87,4 +96,24 @@ public ResponseEntity<GetCaseReferenceResponse> updateExternalInterest(@PathVari
         externalInterestService.updateExternalInterest(caseUUID, interestUUID, extensionData);
         return ResponseEntity.ok(GetCaseReferenceResponse.from(caseUUID, caseDataService.getCaseRef(caseUUID)));
     }
+
+    @PostMapping(path = "/case/{caseUUID}/stage/{stageUUID}/actions/suspension")
+    public ResponseEntity<GetCaseReferenceResponse> applyCaseSuspension(@PathVariable UUID caseUUID,
+                                                                        @PathVariable UUID stageUUID,
+                                                                        @RequestBody ActionDataSuspendDto suspendDto) {
+
+        String caseRef = suspensionService.suspend(caseUUID, stageUUID, suspendDto);
+        return ResponseEntity.ok(GetCaseReferenceResponse.from(caseUUID, caseRef));
+    }
+
+
+    @DeleteMapping(path = "/case/{caseUUID}/stage/{stageUUID}/actions/suspension/{suspensionUUID}")
```

Comment:
Not sure delete mapping is the right mapping here, when we unsuspend a case we're not deleting any records as far as I can tell. If we are just updating the data_suspension_removed then PUT probably more suited here since we are updating a record

Reply:
Development hangover... I've updated to a PUT request